### PR TITLE
[Doc] Updated release docs with gh-release instructions

### PR DESCRIPTION
**Pull request by macrigiuseppe**
Signed-off-by: Giuseppe Macri <gmacri@uber.com>

### CODE REVIEW

File: contributing/DEVELOPERS.md [hunk 1]
```diff
@@ -250,17 +250,26 @@ Netlify is connected to the following github triggers:
...
+#### Requirements
+To prepare a new release you need the following tool:
+- [gh-release](https://www.npmjs.com/package/gh-release): this tool facilitates the creation of a new git tag (using package.json version number) and a github release (different from npm release)
+
+Setup ```gh-release``` with your github api token ([instructions](https://www.npmjs.com/package/gh-release#command-line-interface))
+
+### Push a new release
+In order to publish a new version of kepler.gl a developer must perform the following steps:
 1. Update __package.json__ file with the new version value.
 2. Update each of the example folder package.json kepler.gl dependency with the newer version specified during step 1.
```
Comment (heshan0131) [inline contributing/DEVELOPERS.md:267]: example version should be updated after the release for build to succeed with script `npm run example-version`

File: contributing/DEVELOPERS.md [hunk 2]
```diff
@@ -250,17 +250,26 @@ Netlify is connected to the following github triggers:
...
+
+#### Requirements
+To prepare a new release you need the following tool:
+- [gh-release](https://www.npmjs.com/package/gh-release): this tool facilitates the creation of a new git tag (using package.json version number) and a github release (different from npm release)
+
+Setup ```gh-release``` with your github api token ([instructions](https://www.npmjs.com/package/gh-release#command-line-interface))
+
+### Push a new release
+In order to publish a new version of kepler.gl a developer must perform the following steps:
 1. Update __package.json__ file with the new version value.
```
Comment (heshan0131) [inline contributing/DEVELOPERS.md:266]: run `npm version major | minor | patch`

File: contributing/DEVELOPERS.md [hunk 3]
```diff
@@ -250,17 +250,26 @@ Netlify is connected to the following github triggers:
...
 2. Update each of the example folder package.json kepler.gl dependency with the newer version specified during step 1.
 3. Update __CHANGELOG.md__ with the latest commit changes.
```
Comment (heshan0131) [inline contributing/DEVELOPERS.md:268]: You can print the commits with `git log --pretty=oneline --abbrev-commit`

**Comment by macrigiuseppe** [comment]
@heshan0131 updated with your feedabck

File: contributing/DEVELOPERS.md [hunk 4]
```diff
@@ -250,17 +250,29 @@ Netlify is connected to the following github triggers:
...
+In order to publish a new version of kepler.gl a developer must perform the following steps:
+1. Update __package.json__ file with the new version value. Run ```npm version major | minor | patch``` to update version accordingly.
+2. Update __CHANGELOG.md__ with the latest commit changes. Print commits with ```git log --pretty=oneline --abbrev-commit```
+3. Create a new PR for review.
+4. Once the PR is reviewed and merged, pull the latest changes locally.
+5. Run ```gh-release```: this command will create a new Github Release with the new updated CHANGELOG.md section.
+6. Once the new Github Release is created, Github will automatically trgger a new Github Action flow that will automatically build and publish the new package version to NPM registry.
+
+__After Release is completed and pushed__
+* Update each of the example folder package.json kepler.gl dependency with the newer. This step is required after the enw version is published otherwise it would fail.
```
Comment (heshan0131) [inline contributing/DEVELOPERS.md:274]: run `npm run example-version`

**Pull request merged**